testplan/report/testing/base.py

```diff
@@ -913,15 +882,17 @@ def status(self, new_status):
         self._status = new_status
 
     @property
-    def runtime_status(self):
+    def runtime_status(self) -> str:
         """
-        Used for interactive mode, the runtime status of a testcase may be one
-        of ``RuntimeStatus``.
+        The runtime status is used in interactive mode and stands for
+        whether a particular entry is READY, WAITING, RUNNING,
+        RESETTING, FINISHED, or NOT_RUN.
         """
         return self._runtime_status
 
+    # In principle, this does not have to change
```

Comment:
what does this comment mean?

Reply:
Good question, I think I was looking into the runtime status early in development and left some comments for myself as markers. Will remove.